inspirehep/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -1007,9 +1035,12 @@
             }
         },
         "page_nr": {
-            "type": "string",
-            "description": "FIXME: shall it be integer perhaps?",
-            "title": "Number of pages"
+            "type": "array",
```

Comment:
Since this is an array now, I think the literature submission has a potential problem @jmartinm 

```shell
inspirehep/modules/deposit/workflows/literature.py
328:        if 'page_nr' not in metadata:
333:                        metadata['page_nr'] = int(pages[1]) - int(pages[0]) + 1
```